src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -3479,4 +3480,42 @@ WindowID MessageHelper::ExtractWindowIdFromSmartObject(
   return mobile_apis::PredefinedWindows::DEFAULT_WINDOW;
 }
 
+void MessageHelper::AddDefaultParamsToTireStatus(
+    ApplicationSharedPtr app, smart_objects::SmartObject& response_from_hmi) {
+  const utils::SemanticVersion last_version_with_mandatory_params(7, 1, 0);
+
+  if (!app) {
+    SDL_LOG_ERROR("Application not found");
+    return;
+  }
+
+  if (app->msg_version() > last_version_with_mandatory_params) {
+    SDL_LOG_DEBUG(
+        "Tire status parameters are "
+        "non-mandatory for this app version, no need in default values");
+    return;
+  }
+
+  smart_objects::SmartObject& tire_status =
+      response_from_hmi[strings::msg_params][strings::tire_pressure];
+
+  if (!tire_status.keyExists("pressureTelltale")) {
```

Comment:
@ychernysheva I believe all these parameter names are already declared in the smart object keys file. Try to avoid hardcoding and reuse existing constants here and in `tires` container

Reply:
Done in f40c97b